src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,18 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  //if(_t==getStartTime())
+  if(_t==mWaypoints[0].t)
+  {
+      const auto segmentTime = mWaypoints[1].t - mWaypoints[0].t;
+      const auto alpha = (mWaypoints[1].t - mWaypoints[0].t) / segmentTime;
```

Comment:
It looks like `alpha` will always be `1` because the numerator and denominator are both `mWaypoints[1].t - mWaypoints[0].t`. What am I missing/